graphql/project_settings_resolver.go

```diff
@@ -21,11 +21,13 @@ func (r *projectSettingsResolver) Aliases(ctx context.Context, obj *restModel.AP
 
 // GithubWebhooksEnabled is the resolver for the githubWebhooksEnabled field.
 func (r *projectSettingsResolver) GithubWebhooksEnabled(ctx context.Context, obj *restModel.APIProjectSettings) (bool, error) {
-	hook, err := model.FindGithubHook(utility.FromStringPtr(obj.ProjectRef.Owner), utility.FromStringPtr(obj.ProjectRef.Repo))
+	owner := utility.FromStringPtr(obj.ProjectRef.Owner)
+	repo := utility.FromStringPtr(obj.ProjectRef.Repo)
+	hasApp, err := evergreen.GetEnvironment().Settings().HasGitHubApp(ctx, owner, repo, nil)
 	if err != nil {
-		return false, InternalServerError.Send(ctx, fmt.Sprintf("Database error finding github hook for project '%s': %s", *obj.ProjectRef.Id, err.Error()))
+		return false, InternalServerError.Send(ctx, fmt.Sprintf("Error GitHub app installation for project '%s' in '%s/%s': '%s'", utility.FromStringPtr(obj.ProjectRef.Id), owner, repo, err.Error()))
```

Comment:
nit: "error github app installation" doesn't work as a sentence

Reply:
shouldn't this just say issue getting github app though?  Seems like the error _wouldn't_ be about installation, since we don't error in that case (same with the grip statement below)

Reply:
i meant to write error *verifying* app installation